test/models/MongoDateTimeFormatsSpec.scala

```diff
@@ -30,7 +31,7 @@ class MongoDateTimeFormatsSpec extends FreeSpec with MustMatchers with OptionVal
     val dateMillis = 1517443200000L
 
     val json = Json.obj(
-      "$date" -> dateMillis
+      "$date" -> Json.obj("$numberLong" -> dateMillis.toString)
```

Comment:
this might seem like a breaking change, but I've tested it locally and this change seems to be backwards compatible (new version of the service can read records created by the old and vice versa),
I think the new library exposes BSON dates in a slightly different format